plugin/src/main/java/org/openrewrite/gradle/DelegatingProjectParser.java

```diff
@@ -125,6 +125,8 @@ protected URL jarContainingResource(String resourcePath) {
                     resourcePath = resourcePath.substring(0, indexOfBang);
                 }
                 return new URI(resourcePath).toURL();
+            } else if (resourcePath.startsWith("file:")) {
+                return new URI(resourcePath.substring(0, resourcePath.lastIndexOf("/main/") + 6)).toURL();
```

Comment:
What is this?

Reply:
That is an unrelated change I should commit separately. Without this I cannot debug the tests using the debugger...